lib/context_adapter_communication.js

```diff
@@ -183,6 +183,22 @@ function checkUpdateContextRequest(request, callback) {
 }
 
 /**
+ * Returns true if the updateContext request corresponds to a polling request from the button
+ * @param {object} request The received request
+ * @return {boolean} true if the updateContext request corresponds to a polling request from the button,
+ *  false otherwise
+ */
+function isPollingUpdateContext(request) {
+  if (request && request.payload && request.payload.contextElements && request.payload.contextElements[0]) {
+    if (caHelper.getAttributeValue(
+        request.payload.contextElements[0].attributes, caConfig.BUTTON_ENTITY.CA_LAST_OPERATION) === 'P') {
```

Comment:
Yeah, for the time being only one request (info) will come in each updateContext, as far as I know? :)